pkgs/by-name/aw/awatcher/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, pkg-config
+, gnomeSupport ? true
+, kwinSupport ? true
```

Comment:
```suggestion
```
there are no extra dependencies for those, so we can just compile them all the time